### DISCUSSION TITLE
Libre2 + MiaoMiao setup

### DISCUSSION THREAD

**Discussion by davidreis97** [NightscoutFoundation/xDrip#1925] [Q&A]
Hi all, I am trying to set up a Libre2 sensor with MiaoMiao and XDrip+. I'm following [the official miaomiao guide](https://miaomiao.cool/pages/how-to-use-miaomiao-with-xdrip) but unfortunately, I'm getting stuck in the "Receiving Data from Tomato collector" step and a Toast pops up saying something about needing OOP2. This is strange since I've been using this setup without any issues in the past, but I had to reinstall XDrip+ and now I can't seem to get it to work. Can this be because the sensor is not new?

**Comment by tzachi-dar** [comment]
Do you have an oop1 on your phone?
Has anything changed, beside reinstalling xDrip?

In any case,
https://www.minimallooper.com/post/how-to-setup-freestyle-libre-2-and-oop2-to-use-a-native-bluetooth-connection-in-xdrip
explains how to use libre2 with xDrip without extra hw, you might find that
it solves your problem.

Thanks
Tzachi
